application/config/locales/connectors/dataverse/en.yml

```diff
@@ -80,7 +80,10 @@ en:
         field_type_label: "Type"
         unnamed_dataverse_text: "Unnamed Dataverse"
       search_items:
+        button_submit_text: "Search"
         content_no_items_found_text: "No items found."
+        field_search_label: "Search items"
+        field_search_placeholder: "Filter items"
```

Comment:
These new items should be consistent. All search or all filter.

Reply:
fixed